Gaijinizer patch/CommonEvents_131.txt

```diff
@@ -17,18 +17,18 @@ you suck via other Daji's! ♡#DL
 > CONTEXT: CommonEvents_131/87/Dialogue/enemy17
 Hohoho...♡♡ Don't struggle so hard...
 
-You can't escape now...♡#DL
+You can't escape anymore... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 無駄な足掻きをする貴方様もとっても可愛らしいですわ♡
 
 もっと快楽に歪んだ顔を見せて…♡
 > CONTEXT: CommonEvents_131/91/Dialogue/enemy18
-It's so cute that you're struggling
-uselessly, too♡
+It's so cute that you're pointlessly
+struggling...♡
 
-Show me a more pleasured and contorted face...♡#DL
+Show more of your face perverted in pleasure...♡#Ron
 > END STRING
```

Comment:
Note

Reply:
Mind if I change "me" to "us", since that dialogue pops up when all of them are there?

Reply:
Definitely, sounds great 👍